ecs-logs-collector.sh

```diff
@@ -172,7 +172,7 @@ try_set_instance_infodir() {
       # Put logs into a directory for this instance.
       infodir="${infodir}/${instance_id}"
       info_system="${infodir}/system"
-      echo "$instance_id" | $info_system/instance-id.txt
+      echo "$instance_id" | "$info_system"/instance-id.txt
```

Comment:
Looks like this was missed in the original review of this section:
```suggestion
      mkdir -p "$info_system"
      echo "$instance_id" > "$info_system"/instance-id.txt
```